sql/flights.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS flights;
+
+CREATE TABLE flights (
+    id		INT DEFAULT 1,
+    link	VARCHAR(250),
+    visible     BOOLEAN DEFAULT FALSE,
```

Comment:
I was thinking if they don't want to change the link but just want to hide it during non break times so that people's info isn't easily visible, and after breaks the sheets are old/unnecessary info that doesn't need to be displayed. And I thought it'd be better to have a 'check again later' page than a blank table. idk

Reply:
Ah I meant the SQL table, not the spreadsheet. I think it would be fine to just delete the link when it's out of season.

Reply:
I was thinking like during season the sheet is visible and off season the sheet is hidden and for the next season the secretary can just make the sheet visible again and delete everything on the sheet. Like they don't have to input the link again if they want to reuse the same sheet for its headers or whatnot

Reply:
Sure, I don't feel super strongly about it, but it doesn't seem like it would be too problematic to just have them look up the link of the spreadsheet again if they have to reset the sheet anyways.